pages/popup.html

```diff
@@ -7,10 +7,12 @@
   </head>
   <body>
     <div class="header">Fill out the info below and click save to clip this page to Archivist</div>
-    <form>
-        <label for="title">Title</label><input type="text" id="title" name="title" placeholder="Enter Title">
-        <label for="author">Author</label><input type="text" id="author" name="author" placeholder="Enter Author">
-        <input type="button" value="Clip Page">
+    <form id="metadata-form">
+        <template id="input-template">
+            <label></label><input>
+        </template>
+        <input id="save-page-btn" type="button" value="Save Page">
```

Comment:
Isn't it proper to use `<input />` over `<input>`?  I dunno so I'm just double checking.